src/scripting/game_lua_kernel.cpp

```diff
@@ -3730,6 +3730,19 @@ int game_lua_kernel::intf_get_color_adjust(lua_State *L)
 	return 0;
 }
 
+int game_lua_kernel::intf_screen_fade(lua_State *L)
+{
+	if(game_display_) {
+		auto vec = lua_check<std::vector<uint8_t>>(L, 1);
+		if(vec.size() != 4) {
+			return luaL_error(L, "screen fade colour must be an array of 4 integers");
```

Comment:
```suggestion
			return luaW_type_error(L, 1, "array of 4 integers");
```

This makes the error be something like:

`bad argument #1 to 'screen_fade' (array of 4 integers expected, got nil)`